va/va.go

```diff
@@ -498,6 +499,11 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 	// io.LimitedReader will silently truncate a Reader so if the
 	// resulting payload is the same size as maxResponseSize fail
 	if len(body) >= maxResponseSize {
+		// If the body isn't UTF-8, it will cause problems when marshaling the
+		// response as a protocol buffer. If so, just skip showing the body.
+		if !utf8.Valid(body) {
```

Comment:
This is the nuclear approach, which tbh I'm pretty fine with, but you think there might be any value in having a loop and just chopping a byte off the end of the body until it is UTF-8 valid (which would cover the case where we've truncated a codepoint by one byte)?

Reply:
+1 - to this feedback. I appreciate that this would be more complex but when folks fall into the generic "Invalid response" bucket its a bit harder to suss out the root cause and often ends with someone tagging one of us in the community forum to check the VA logs.